src/content/docs/network-performance-monitoring/get-started/network-performance-monitoring-best-practices.mdx

```diff
@@ -0,0 +1,292 @@
+
+
+In the [common architecture example](#common-architecture) above, the following considerations are in place:
+
+* Each geographical location has its own local containers used to collect and forward data to New Relic.
+* `DC_01` has three containers:
+
+    * Container 1: SNMP polling
+    * Container 2: NetFlow v5 collection
+    * Container 3: Syslog collection
+
+    <Callout variant="important">
+    In `DC_01` there is a Class B private subnet (/16). Care needs to be taken to split up discovery targets into manageable subnet sizes to ensure the job has time to complete.
+    </Callout>
+
+* `OFFICE_01` uses a single container for SNMP polling with discovery against a /24 subnet.
+
+* `DC_02` has three containers, all dedicated to different templates in network flow collection:
+
+    * Container 1: NetFlow v9
+    * Container 2: IPFIX
+    * Container 3: sFlow
+
+    <Callout variant="important">
+    In `DC_02`, there is an even larger Class A private subnet (/8); however, since there is no need for a discovery in this location there are no concerns about scaling the jobs. Depending on flows-per-second, there may be a need in the future to scale out into additional containers. In a situation like this, you simply need to configure your network flow exporters to target different ports on the Docker host.
+    </Callout>
+
+## Maintaining your deployment [#maintaining-deployment]
+
+After initial installation, there are various techniques that are used to maintain the network monitoring observability footprint. Integrating configuration file changes with tools like Ansible are common, as is building GitOps pipelines around the architecture to support versioning and “guest” options where external teams can submit changes for review.
+
+The most common need for ongoing maintenance is in keeping the list of target devices accurate. There are three options you can take to support this goal:
+
+* (Recommended) Option 1: [Automatic discovery](#automatic-discovery)
+* Option 2: [Manual discovery](#manual-discovery)
+* Option 3: [Manual device addition](#manual-device-addition)
+
+### Automatic discovery (preferred) [#automatic-discovery]
+
+Automatic discovery is the process used by KTranslate to scan a target list of IP addresses and/or ranges, perform a liveness probe, and then run a basic SNMP walk of the MIB-2 System MIB to attempt matching the device to a known SNMP profile.
+
+The container has embedded container runtime flags (`-snmp_discovery_min` and `-snmp_discovery_on_start`) that allow administrators to build a schedule of recurring SNMP discovery events. This automates discovery jobs against the targets from the discovery section in the configuration file, and then automatically update the file with new devices along with refreshing the service so it accepts said changes. 
+
+This is the native configuration pattern found if you follow the guided installation through the New Relic UI.
```

Comment:
What is "the native configuration pattern" in reference to? Is it referencing the embedded container runtime flags, or something else?